include/tvm/relay/adt.h

```diff
@@ -91,13 +91,9 @@ class PatternVar;
 /*! \brief PatternVar container node */
```

Comment:
@tqchen Do we want to move `make` in `PatternWildcardNode` to `PatternWildcard`? There is already a default constructor for `PatternWildcard`.

Reply:
Yes, We can allow multiple constructors as long as there is no ambiguity

Reply:
This one would be ambiguous if we create another constructor `PatternWildcard()`

Reply:
I see, in this case we should simply override the default constructor to the new behavior(do not use the original constructor). We would likely need to add copy/move constructor and operator= overloads though in this case as well.

Reply:
done. PTAL